src/HIE/Bios/Environment.hs

```diff
@@ -167,6 +167,20 @@ addCmdOpts cmdOpts df1 = do
        liftIO $ hPutStrLn stderr "cannot set package flags with :seti; use :set"
     -}
 
+makePathsAbsolute :: FilePath -> DynFlags -> DynFlags
+makePathsAbsolute work_dir df = 
+  mapOverIncludePaths (prependIfNotAbsolute work_dir)
+  $ df
+    { importPaths = map (prependIfNotAbsolute work_dir) (importPaths df)
+    -- , libraryPaths  = map (prependIfNotAbsolute work_dir) (libraryPaths df)
```

Comment:
What is this commented out stuff?

Reply:
nothing in particular, just saw other `FilePath`'s that could be made absolute if they are relative. 
Just thought it might be good, but we didnt do it before and now it is trivial to fix it if there are more problems with relative filepaths.